packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -133,36 +136,42 @@ This method of configuring a custom domain is recommended because Okta manages y
 
-> **Note:** Some orgs support [DNSSEC (Domain Name System Security Extensions)](https://datatracker.ietf.org/doc/html/rfc9364) and need to point their custom domain to `example.okta-dnssec.com`, instead of `example.okta.com`. This value appears for all new and existing custom domains and allows you to be DNSSEC compliant. See [Supportability for DNSSEC in Okta Organizations](https://support.okta.com/help/s/article/supportability-for-dnssec-in-okta-organizations?language=en_US).
+> **Note:** Some orgs support [DNSSEC](https://datatracker.ietf.org/doc/html/rfc9364) and need to point their custom domain to `example.okta-dnssec.com`, instead of `example.okta.com`. This value appears for all new and existing custom domains and allows you to be DNSSEC compliant. See [Supportability for DNSSEC in Okta Organizations](https://support.okta.com/help/s/article/supportability-for-dnssec-in-okta-organizations?language=en_US).
 
 1. On the **Update your DNS** page of the configuration wizard, copy the values of the **Host** and **Value** columns into a text file.
+
 2. Sign in to your Domain Name registrar and locate the option to modify your DNS records.
-3. Add a TXT record and paste the value that you copied from the **Host** column into the appropriate field, for example, the **Name** or **Host** field.
+
+3. Add a TXT record and paste the value that you copied from the **Host** column into the appropriate field. For example, paste the value into the **Name** or **Host** fields.
 
     > **Note**: Depending on your domain provider, you may only need to enter `_acme-challenge.login` rather than `_acme-challenge.login.example.com`. If your domain provider doesn't support the value that you enter, verification fails and your custom URL domain configuration is incomplete.
     >
     > You can perform a DNS lookup of your `_acme-challenge` DNS record to verify that it's correctly configured. For example, you might use Google's [Dig](https://toolbox.googleapps.com/apps/dig/) tool to check your `_acme-challenge.login.example.com` DNS record.
 
 4. Paste the value that you copied from the **Value** column into the appropriate field, for example, the **Record** or **Value** field.
+
 5. Repeat Steps 3 and 4 for the CNAME record.
-6. Wait for the DNS record to propagate (typically one to five minutes, but it may take longer), and then return to Okta and click **Next** to prove to Okta that you have rights to use the domain name.
+
+6. Wait for the DNS record to propagate, typically one to five minutes, but it may take longer. Then, return to Okta and click **Next** to prove to Okta that you have the rights to use the domain name.
```

Comment:
```suggestion
6. Wait for the DNS record to propagate. Typically, this takes one to five minutes (but it may take longer). Then, return to Okta and click **Next** to prove to Okta that you have the rights to use the domain name.
```